dev/doc/INSTALL.dune.md

```diff
@@ -14,28 +13,20 @@ Usually, using the latest version of Dune is recommended, see the
 first line of the `dune-project` file for the minimum required
 version.
 
-If you set `COQ_USE_DUNE=1`, then you don't need to explicitly add `-f
-Makefile.dune` in any of the commands below. However, you will then
-need an explicit `-f Makefile.make` if you want to use one of the
-legacy targets.
-
-It is strongly recommended that you use the helper targets available
-in `Makefile.dune`, `make -f Makefile.dune` will display help. Note
-that dune will call configure for you if needed, so no need to call
-`./configure` in the regular development workflow.
-
 4 common operations targets are:
 
-- `make -f Makefile.dune check` : build all ml targets as fast as possible
-- `make -f Makefile.dune world` : build a complete Coq distribution
-- `dune exec -- dev/shim/coqtop-prelude` : build and launch coqtop + prelude [equivalent to `make states`].
+- `dune build @check` : build all ml targets as fast as possible
+- `dune build` : build a complete Coq distribution
+- `dune exec -- dev/shim/coqtop-prelude` : build and launch coqtop + prelude [equivalent to `make -f Makefile.dune states`].
 - `dune build $target`: where `$target` can refer to the build
   directory or the source directory [but will be placed under
   `_build`]
 
-`dune build @install` will build all the public Coq artifacts; `dune
-build` will build all the targets in the workspace, including tests
-and documentation (so this is usually not what you want).
+Optionally, you can view the helper targets available in `Makefile.dune`;
+`make -f Makefile.dune` will display help. The command invokes various
+`dune` commands for you with the right arguments. Note that dune will
+call configure for you if needed, so no need to call `./configure` in
+the regular development workflow.
```

Comment:
IIUC you must not call `./configure` manually if you want to build with Dune.

Reply:
Yes, that is correct. Let me double-check the wording.